### ISSUE TITLE
Issue with Processes subscriptions meta id

### ISSUE THREAD

**Issue by egregors** [logux/backend-test#7]
test: ✖ 11 Processes subscriptions

All seems good, but I return wrong meta id: 6 instead of 5:
```shell
✖ 11 Processes subscriptions

  Server sent {"type":"logux/processed","id":"6 10:2:1 0"}, instead of {"type":"logux/processed","id":"5 10:2:1 0"} action

Test:      /Users/egregors/Documents/GitHub/django/lbt/node_modules/@logux/backend-test/tests/actions.js:41:3
Re-run it: npx @logux/backend-test http://localhost:8000/logux/ 11
```
on this step in the answer you are expecting `"id":"5 10:2:1 0"`, but logux server sent action:
```json
{
  "command": "action",
  "action": {
    "type": "logux/subscribe",
    "channel": "users/10"
  },
  "meta": {
    "id": "6 10:2:1 0",
    "time": 6,
    "subprotocol": "1.0.0",
    "reasons": [],
    "server": "server:1"
  },
  "headers": {}
}
```
with `'id': '6 10:2:1 0'`. What could be wrong here?

Full log: 
```shell
2020-07-16 10:44:52,762 - logux.views - DEBUG - got auth cmd: {'command': 'auth', 'authId': 'i92rJ6WhRaArUbr7oQpyU', 'userId': '10', 'token': '10:good', 'headers': {}, 'cookie': {}, 'subprotocol': '1.0.0'}
2020-07-16 10:44:52,763 - logux.views - DEBUG - {'answer': 'authenticated', 'subprotocol': '1.0.0', 'authId': 'i92rJ6WhRaArUbr7oQpyU'}
[16/Jul/2020 10:44:52] "POST /logux/ HTTP/1.1" 200 88
2020-07-16 10:44:52,773 - logux.views - DEBUG - got auth cmd: {'command': 'auth', 'authId': 'DOpuTRot3UQabWQsZhbqb', 'userId': '10', 'token': '10:good', 'headers': {}, 'cookie': {}, 'subprotocol': '1.0.0'}
2020-07-16 10:44:52,773 - logux.views - DEBUG - {'answer': 'authenticated', 'subprotocol': '1.0.0', 'authId': 'DOpuTRot3UQabWQsZhbqb'}
[16/Jul/2020 10:44:52] "POST /logux/ HTTP/1.1" 200 88
2020-07-16 10:44:52,780 - logux.views - DEBUG - got auth cmd: {'command': 'auth', 'authId': 'uWIt1eMgqp9nn23k1_Mue', 'userId': '10', 'token': '10:good', 'headers': {}, 'cookie': {}, 'subprotocol': '1.0.0'}
2020-07-16 10:44:52,780 - logux.views - DEBUG - {'answer': 'authenticated', 'subprotocol': '1.0.0', 'authId': 'uWIt1eMgqp9nn23k1_Mue'}
[16/Jul/2020 10:44:52] "POST /logux/ HTTP/1.1" 200 88
2020-07-16 10:44:52,788 - logux.views - DEBUG - got action: {'command': 'action', 'action': {'type': 'logux/subscribe', 'channel': 'users/10'}, 'meta': {'id': '1 10:1:1 0', 'time': 1, 'subprotocol': '1.0.0', 'reasons': [], 'server': 'server:1'}, 'headers': {}}
2020-07-16 10:44:52,788 - logux.views - DEBUG - got subscription for channel: users/10
2020-07-16 10:44:52,790 - logux.views - DEBUG - {'answer': 'approved', 'id': '1 10:1:1 0'}
2020-07-16 10:44:52,790 - logux.views - DEBUG - {'answer': 'action', 'id': '1 10:1:1 0', 'action': {'type': 'users/name', 'payload': {'userId': '10', 'name': 'Name'}}, 'meta': {'clients': ['10:1']}}
2020-07-16 10:44:52,790 - logux.views - DEBUG - {'answer': 'processed', 'id': '1 10:1:1 0'}
[16/Jul/2020 10:44:52] "POST /logux/ HTTP/1.1" 200 241
2020-07-16 10:44:52,798 - logux.views - DEBUG - got action: {'command': 'action', 'action': {'type': 'users/name', 'payload': {'userId': '10', 'name': 'Name'}}, 'meta': {'id': '4 10:1:1 0', 'time': 4, 'subprotocol': '1.0.0'}, 'headers': {}}
2020-07-16 10:44:52,801 - logux.views - DEBUG - {'answer': 'resend', 'id': '4 10:1:1 0', 'channels': ['users/10']}
2020-07-16 10:44:52,802 - logux.views - DEBUG - {'answer': 'approved', 'id': '4 10:1:1 0'}
2020-07-16 10:44:52,802 - logux.views - DEBUG - {'answer': 'processed', 'id': '4 10:1:1 0'}
[16/Jul/2020 10:44:52] "POST /logux/ HTTP/1.1" 200 157
2020-07-16 10:44:52,808 - logux.views - DEBUG - got action: {'command': 'action', 'action': {'type': 'logux/subscribe', 'channel': 'users/10'}, 'meta': {'id': '6 10:2:1 0', 'time': 6, 'subprotocol': '1.0.0', 'reasons': [], 'server': 'server:1'}, 'headers': {}}
2020-07-16 10:44:52,808 - logux.views - DEBUG - got subscription for channel: users/10
2020-07-16 10:44:52,810 - logux.views - DEBUG - {'answer': 'approved', 'id': '6 10:2:1 0'}
2020-07-16 10:44:52,810 - logux.views - DEBUG - {'answer': 'action', 'id': '6 10:2:1 0', 'action': {'type': 'users/name', 'payload': {'userId': '10', 'name': 'Name'}}, 'meta': {'clients': ['10:2']}}
2020-07-16 10:44:52,810 - logux.views - DEBUG - {'answer': 'processed', 'id': '6 10:2:1 0'}
[16/Jul/2020 10:44:52] "POST /logux/ HTTP/1.1" 200 241
```

**Comment by ai** [comment]
Fixed 17b0778

**Comment by ai** [comment]
Released in 4.0.6

**Comment by ai** [comment]
Now tests are looking the real action ID from the log instead of using hard-coded IDs
